I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 21, in script
    jump map_label
ScriptError: could not find label 'map_label'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 21, in script
    jump map_label
  File "/home/gkiss/Misc/renpy-6.99.14.3-sdk/renpy/ast.py", line 1600, in execute
    rv = renpy.game.script.lookup(target)
  File "/home/gkiss/Misc/renpy-6.99.14.3-sdk/renpy/script.py", line 873, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'map_label'.

Linux-4.13.0-38-generic-x86_64-with-debian-stretch-sid
Ren'Py 6.99.14.3.3347
RenpyTest 1.0
Tue Apr 17 07:00:06 2018
